engine/src/cmd/ai/pythonai.cpp

```diff
@@ -23,10 +23,11 @@
  */
 
 
+#define PY_SSIZE_T_CLEAN
 #include <boost/version.hpp>
 #include <boost/python/class.hpp>
 
-#include <Python.h>
+#include <Python.h> // TODO: Remove direct inclusion of this file?
```

Comment:
@BenjamenMeyer with regard to Python.h and `#include`ing `PY_SSIZE_T_CLEAN`: I am following what @LifWirser recommended in #700 , which in turn is taken from python.org documentation I believe.

Reply:
Could we solve that by including `pythonai.h` first and then doing the others? That file should also have things in the correct order too. Just looking to see how we can reduce duplication. If we can't, then not a problem.

Reply:
Hmm. It will only save us one line in each location. And it will be harder for people reading or reviewing the code to verify whether or not it conforms to @LifWirser 's and python.org's recommendations.

I vote we not bother, frankly.